README.md

```diff
@@ -59,6 +59,7 @@ googleAnalytics = "" # Enable Google Analytics by entering your tracking id
 [Params]
   subtitle = "Just another site" # Subtitle of your site. Used in site header
   description = "John Doe's Personal blog about everything" # Site description. Used in meta description
+  #copyright = "John Doe"  # copyright holder, otherwise will use site title
```

Comment:
It's not a big deal at all, really, but... Can you fix two spaces between param and comment? Thank you!

Reply:
Not at all--sorry for the 2x sloppiness!